docs/data/data-grid/master-detail/FullWidthDetailPanel.tsx

```diff
@@ -205,7 +204,6 @@ export default function FullWidthDetailPanel() {
       <DataGridPro
         columns={columns}
         rows={rows}
-        pinnedColumns={{ left: [GRID_DETAIL_PANEL_TOGGLE_FIELD] }}
```

Comment:
This was used to keep toggle buttons on the left.
After it is removed, pinning a column to the left will result in the following
<img width="745" alt="pinning" src="https://github.com/user-attachments/assets/9e7a2d77-08b0-49f9-b4ab-cdc88965c946">

You need to use API to [control the pinned](https://mui.com/x/react-data-grid/column-pinning/#controlling-the-pinned-columns) columns and always keep the `GRID_DETAIL_PANEL_TOGGLE_FIELD` at the most left column

Reply:
got that

Reply:
@arminmeh updated the PR, please have a look when you get sometime. Thank you

Reply:
Alternatively, `initialState` could be used.